core/domain/exp_domain.py

```diff
@@ -556,7 +557,7 @@ def __init__(
             states_schema_version, init_state_name, states_dict,
             param_specs_dict, param_changes_list, version,
             auto_tts_enabled, correctness_feedback_enabled,
-            created_on=None, last_updated=None):
+            proto_size_in_bytes=0, created_on=None, last_updated=None):
```

Comment:
Why do we need a default value here?

Reply:
I am assigning this as a default value to zero because let's suppose whenever anyone wants to create a new exploration I don't want the developer to pass the value, rather the size will be calculated under the hood and added to the exploration as an attribute. I should have a called the size calculation function at the end of the `def __init__()` function. 

Do you think the idea of abstracting the exploration size calculation from the developer is a good one? 

It cannot be `None` because either size of the exploration is something integer, for example `100` or zero. `None` cannot fit here.